modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/constants.js

```diff
@@ -3,6 +3,8 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+export const ACCOUNT_ENTRY_ID_DEFAULT = 0;
+
```

Comment:
I think the empty line can be removed

Reply:
and is is correct ID_DEFAULT instead of DEFAULT_ID?

Reply:
From BE